### PR TITLE
KEYCLOAK-17302 - exclude for Remote testsuite

### PR DESCRIPTION
**Pull request by lhanusov**

### CODE REVIEW

**Comment by lhanusov** [comment]
@pdrozd @vramik I excluded this unstable test and created minor task: KEYCLOAK-17659, where it can be fixed.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java [hunk 1]
```diff
@@ -219,7 +219,7 @@ public void testPOSTAuthenticationRequest() {
         client.close();
     }
 
-    @AuthServerContainerExclude(value = AuthServerContainerExclude.AuthServer.QUARKUS, details = "Unstable for Quarkus, review later")
+    @AuthServerContainerExclude(value = {AuthServerContainerExclude.AuthServer.QUARKUS, AuthServerContainerExclude.AuthServer.REMOTE}, details = "Unstable for Quarkus, review later")
```
Comment (vramik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java:222]: Please update `details` message.

**Review by vramik — CHANGES_REQUESTED** [review]
Thank you @lhanusov for the PR.

Can you please explain the unstability? My understanding was that the test fails stable on remote environment due to not configured field in config file (`standalone.xml`).

If it's the case I'd suggest update the `details` message where we can describe why the issue is there, wdyt? Maybe we can also put there a link to the JIRA you've created as well.

note: I believe it's failing on quarkus due to the same reason.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java (hunk 1 above)
Comment (lhanusov) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java]: @vramik I have updated details message. WDYT?

Comment (vramik) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LoginTest.java]: Thanks

**Review by vramik — APPROVED** [review]
Looks OK to me. Thank you @lhanusov

**Pull request merged**